src/App.js

```diff
@@ -1,25 +1,24 @@
-import React, { PureComponent } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import './style/App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import NavBar from './components/NavBar';
+import Home from './pages/home';
+import Calc from './pages/calc';
+import Quote from './pages/quote';
 
-class App extends PureComponent {
-  render() {
-    return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-      </div>
-    );
-  }
+export default function App() {
+  return (
+    <Router>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calc" element={<Calc />} />
+        <Route path="/quote" element={<Quote />} />
```

Comment:
Well done ✔ The website contains Home, Calculator, and Quote 🔥 All links are working as expected 🙌